### PR TITLE
Dol

### PR DESCRIPTION
**Pull request by nomanashraf874**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -12,7 +14,39 @@ public class HelloWorldServlet extends HttpServlet {
...
+    ArrayList<String> greetings = new ArrayList<String>();
+
+    greetings.add("Hello!");
+    greetings.add("Bonjour!");
+    greetings.add("Hola!");
+    greetings.add("Guten tag!");
+    greetings.add("Salve");
+    
+    String json = convertToJson(greetings);
+
```
Comment (kmontgomery) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: nit for readability: remove this newline and the one below.

File: walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md [hunk 2]
```diff
@@ -153,6 +153,11 @@ Modify your servlet file to take the data entered by users in your "contact me"
 form and store it as an entity in Datastore. Get this working before
 you worry about loading the data.
 
+The first time you deploy your application, you may encounter an error like this when
```
Comment (kmontgomery) [inline walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: Are these your changes? I'm not sure they should be included.

**Review by kmontgomery — CHANGES_REQUESTED** [review]
Higher level comment - the description of your PR should be something to tell the reviewer what the code is trying to do so that the reviewer has some context going in.

File: walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md (hunk 2 above)
Comment (nomanashraf874) [reply walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: No they are not not sure where thy came from!

Comment (kmontgomery) [reply walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: Ok, then I think you can go ahead and leave them out of the PR. Run a git command to restore those files.

Comment (nomanashraf874) [reply walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: i just checked the original git files they have those lines as well i don't know why they are showing them as an addition.

Comment (nomanashraf874) [reply walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: so after i do suggest fixes i create a new pull request?

Comment (kmontgomery) [reply walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md:156]: You can just leave them here for now and submit this PR.

**Review by kmontgomery — APPROVED** [review]

**Pull request merged**